plugins/source/aws/resources/services/emr/studio_session_mapping.go

```diff
@@ -0,0 +1,57 @@
+package emr
+
+import (
+	"context"
+	"github.com/aws/aws-sdk-go-v2/service/emr"
+	"github.com/aws/aws-sdk-go-v2/service/emr/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v4/schema"
+	"github.com/cloudquery/plugin-sdk/v4/transformers"
+)
+
+func studioSessionMapping() *schema.Table {
+	tableName := "aws_emr_studio_session_mapping"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/emr/latest/APIReference/API_SessionMappingSummary.html`,
```

Comment:
```suggestion
		Description:         `https://docs.aws.amazon.com/emr/latest/APIReference/API_SessionMappingDetail.html`,
```
Should match the final type the user will access (if not then the call - `API_GetStudioSessionMapping.html`)